lib/secret_keys/encryptor.rb

```diff
@@ -78,6 +108,10 @@ def decrypt(encrypted_str)
     decoded_str.force_encoding(Encoding::UTF_8)
   end
 
+  def encrypted?(str)
+    str.is_a?(String) && str.start_with?(ENCRYPTED_PREFIX)
```

Comment:
Make a class method `encrypted?` with this logic.

```suggestion
    self.class.encrypted?
```